Sources/NetworkProtection/KeyManagement/NetworkProtectionTokenStore.swift

```diff
@@ -29,16 +29,25 @@ public protocol NetworkProtectionTokenStore {
     ///
     func fetchToken() throws -> String?
 
-    /// Obtain the stored auth token.
+    /// Delete the stored auth token.
     ///
     func deleteToken() throws
+
+    /// Convert DDG-access-token to NetP-auth-token
+    ///
+    static func makeToken(from accessToken: String) -> String
```

Comment:
I think the actual function should refer to DDG + NetP and not just the comment. Comments don’t always get read.